client/my-sites/comments/comment-list/index.jsx

```diff
@@ -100,6 +135,16 @@ export class CommentList extends Component {
 					status,
 					toggleBulkEdit: this.toggleBulkEdit,
 				} } />
+				{ null === filteredComments &&
+					<CommentDetailPlaceholder />
+				}
+				{ 0 === size( filteredComments ) &&
+					<EmptyContent
```

Comment:
@dmsnell These two checks are based on https://github.com/Automattic/wp-calypso/pull/14303#pullrequestreview-39636294

> whereas "don't yet now" is `( { userDevices } ) => null === userDevices`
> "has no devices" is `( { userDevices } ) => 0 === size( userDevices )`

Not sure if I got that right though.
Currently the comments are never `null`, so the placeholder is never shown, but I assumed it's a temporary thing. Please correct me if my assumption is wrong though!